lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -272,14 +272,14 @@ import Web.HttpApiData
     ( FromHttpApiData (..), ToHttpApiData (..) )
 
 import qualified Cardano.Crypto.Wallet as CC
+import qualified Cardano.Wallet.Primitive.Types as W
 import qualified Data.Aeson as Aeson
 import qualified Data.Aeson.Types as Aeson
 import qualified Data.ByteString.Lazy as BL
 import qualified Data.HashMap.Strict as HM
 import qualified Data.Map as Map
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as T
-import qualified Cardano.Wallet.Primitive.Types as W
```

Comment:
That change look like a difference in configuration or version of `stylish-haskell` :thinking: ...